cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/status/StatusModule.java

```diff
@@ -111,7 +109,7 @@ private Measurement createMeasurement(CompactionInfo compactionInfo) {
         tags.put("columnfamily", compactionInfo.columnFamily);
         tags.put("taskType", compactionInfo.taskType);
 
-        final Map<String, String> fields = new HashMap<>();
+        final Map<String, String> fields = new HashMap<>(5);
```

Comment:
Would some other structure be a better fit instead of HashMap with fixed length and known keys?